Arduino/libraries/Atmega_servo_response/Atmega_servo_response.h

```diff
@@ -0,0 +1,18 @@
+#ifndef _ATMEGA_SERVO_RESPONSE_H
+#define _ATMEGA_SERVO_RESPONSE_H
```

Comment:
This is a custom header file made for the NodeMCU. It includes the relevant libraries and defines the functions in its respective C++ file.